lib/Indexer/Model/Query/Criteria/ShortNameBeginsWith.php

```diff
@@ -22,6 +25,6 @@ public function isSatisfiedBy(Record $record): bool
             return false;
         }
 
-        return str_starts_with($record->shortName(), $this->name);
+        return str_starts_with(strtolower($record->shortName()), $this->name);
```

Comment:
This place is tempting to try (probably slow) fuzzy matching if `str_starts_with` didn't match:


```suggestion
              if (str_starts_with(strtolower($record->shortName()), $this->name)) {
            return true;
        }

        $regex = '/' . implode('.*', array_map(preg_quote(...), str_split($this->name))) . '/i';

        return preg_match($regex, $record->shortName()) === 1;
  
```
![image](https://github.com/phpactor/phpactor/assets/11404453/40bdc255-2e07-4364-88ae-770759f4b508)